src/scss/layout/_header.scss

```diff
@@ -2,26 +2,47 @@
 
 .site-header {
   // header section
+  position: relative;
   display: flex;
   justify-content: space-between;
-
+  height: rem(80px);
+  
+  &::after {
+    content: '';
+    position: absolute;
+    bottom: 0;
+    left: calc((100vw - 100%) / -2 - 10px);
+    display: block;
+    width: 100vw;
+    height: 1px;
+    background-color: $color__gray-700;
+  }
+  
   // logo link
   &__logo {
     display: flex;
     align-items: center;
+    margin: 0 auto;
     color: $color__white;
     font-weight: bold;
     line-height: 1.05;
     letter-spacing: 1.03px;
     text-transform: uppercase;
     text-decoration: none;
-
+    
+    @include breakpoint('medium') {
+      margin: inherit;
+    }
+    
     // csis logo svg
     .csis-link__logo {
-      margin: rem(20px);
-      margin-right: rem(10px);
+      margin: rem(20px) rem(10px) rem(20px) 0;
```

Comment:
The only margin property the logo needs is a `margin-right`. The rest of the margin is taken care of already by the wrapper class & the align-center property on the site header container.